Formula/tkey-ssh-agent.rb

```diff
@@ -0,0 +1,53 @@
+class TkeySshAgent < Formula
+  desc "SSH agent for use with the TKey security stick"
+  homepage "https://tillitis.se/"
+  url "https://github.com/tillitis/tillitis-key1-apps/archive/v0.0.3.tar.gz"
+  sha256 "df44410c8610460e392e7c128ed40251018af5b97421f9dce5008e3b2bfbb13f"
+  license all_of: ["GPL-2.0-only", "CC0-1.0"]
+
+  depends_on "go" => :build
+
+  on_macos do
+    depends_on "pinentry-mac"
+  end
+
+  on_linux do
+    depends_on "pinentry"
+  end
+
+  resource "signerapp" do
+    url "https://github.com/tillitis/tillitis-key1-apps/releases/download/v0.0.3/signer.bin"
+    sha256 "efec2aa4a703964f19e4079707c5f3f3f3ba3fe06b44833173581b42b0abd258"
+  end
```

Comment:
Is this a binary?

Reply:
It is a binary for uploading to the TKey security stick.